recipe/meta.yaml

```diff
@@ -1,37 +1,40 @@
-{% set version = "13.0.1" %}
+{% set version = "14.0.0" %}
 {% set major_ver = version.split(".")[0] %}
 
 package:
   name: llvm-package
   version: {{ version }}
 
 source:
-  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-{{ version.replace(".rc", "rc") }}.src.tar.xz
-  sha256: ec6b80d82c384acad2dc192903a6cf2cdbaffb889b84bfb98da9d71e630fc834
-  patches:
-    # - patches/intel-D47188-svml-VF.patch    # Fixes vectorizer and extends SVML support
-    # - patches/expect-fastmath-entrypoints-in-add-TLI-mappings.ll.patch # adjusts test added in 10.0.0 for intel-D47188-svml-VF.patch effects
-    # - patches/amd-roc-2.7.0.diff
-    - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
-    # Don't link libLLVM.dylib to llvm-config. Makes it easier for cross compiling
-    - patches/0002-Don-t-link-libLLVM.dylib-to-llvm-config.patch
+  - url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-{{ version.replace(".rc", "rc") }}.src.tar.xz
+    sha256: 4df7ed50b8b7017b90dc22202f6b59e9006a29a9568238c6af28df9c049c7b9b
+    patches:
+      # - patches/intel-D47188-svml-VF.patch    # Fixes vectorizer and extends SVML support
+      # - patches/expect-fastmath-entrypoints-in-add-TLI-mappings.ll.patch # adjusts test added in 10.0.0 for intel-D47188-svml-VF.patch effects
+      # - patches/amd-roc-2.7.0.diff
+      - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
+      # Don't link libLLVM.dylib to llvm-config. Makes it easier for cross compiling
+      - patches/0002-Don-t-link-libLLVM.dylib-to-llvm-config.patch
+  # Workaround for https://github.com/llvm/llvm-project/issues/53281
+  - url: https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-{{ version.replace(".rc", "-rc") }}.tar.gz
```

Comment:
Use this as the source. No need to use two sources.

Reply:
OK, fine by me.